jvm-packages/pom.xml

```diff
@@ -24,9 +24,9 @@
         </developer>
     </developers>
     <scm>
-        <connection>scm:git:git:/github.com/NVIDIA/spark-xgboost.git</connection>
-        <developerConnection>scm:git:ssh://github.com/NVIDIA/spark-xgboost.git</developerConnection>
-        <url>https://github.com/NVIDIA/spark-xgboost</url>
+        <connection>scm:git:git:/github.com/rapidsai/xgboost.git</connection>
```

Comment:
Seems we should change to NVIDIA/spark-xgboost

Reply:
nice catch, updated